LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -15,7 +15,10 @@
 
     <main class="main-content">
 
-        <h2><a href="/LEAF_Request_Portal/admin" class="leaf-crumb-link">Admin</a><i class="fas fa-caret-right leaf-crumb-caret"></i>User access</h2>
+       <!-- LEAF-1839 -- rbs -- fix issue with url misdirect due to hard code url
+       <h2><a href="/LEAF_Request_Portal/admin" class="leaf-crumb-link">Admin</a><i class="fas fa-caret-right leaf-crumb-caret"></i>User access</h2> -->
+        <h2><a href="../admin" class="leaf-crumb-link">Admin</a><i class="fas fa-caret-right leaf-crumb-caret"></i>User access</h2>
```

Comment:
Is there a reason this needed to be a relative link?  Just wanted to see why it was failing.

Reply:
yes it needs a relative link as the hard link was throwing the error because it was used on a page with a diff path

Reply:
OK, just wanted to verify for consistency.  Looks good to me.